openhands/runtime/utils/runtime_build.py

```diff
@@ -260,9 +262,9 @@ def build_runtime_image(
     # Scenario 2: If a Docker image with the exact hash is not found, we will FIRST try to re-build it
     # by leveraging the `generic_runtime_image_name` to save some time
     # from re-building the dependencies (e.g., poetry install, apt install)
-    elif runtime_builder.image_exists(generic_runtime_image_name) and not force_rebuild:
+    if not force_rebuild and runtime_builder.image_exists(generic_runtime_image_name):
```

Comment:
```suggestion
    elif not force_rebuild and runtime_builder.image_exists(generic_runtime_image_name):
```

Reply:
If the first IF results in true, it returns. I don't see why there should be an "elif"?

Reply:
I think this is more like a style preference i have... to make things look more like a "switch". Though it is just nits, feel free to ignore it.